commit 098ad3facb3d76fe4476424f1b7bf3c9c2c1b142
Author: Nicolas Fella <nicolas.fella@gmx.de>
Date:   Fri Jan 17 16:46:19 2025 +0100

    Add missing includes
    
    (cherry picked from commit 9df8a685aa2d7ed1e9e633dda96d141c591f398e)

diff --git a/cursors/src/kcursorgen/main.cpp b/cursors/src/kcursorgen/main.cpp
index 96d5e2ec..b1a284f4 100644
--- a/cursors/src/kcursorgen/main.cpp
+++ b/cursors/src/kcursorgen/main.cpp
@@ -8,6 +8,7 @@
 
 #include <QCommandLineParser>
 #include <QCoreApplication>
+#include <QDebug>
 #include <QString>
 
 int main(int argc, char **argv)
